CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - [Python] Reuse the action cucumber/action-publish-pypi in release ([#147](https://github.com/cucumber/tag-expressions/pull/147))
 - [Ruby] Fixed up remaining simple cops and began to reduce complexity of code ([#158](https://github.com/cucumber/tag-expressions/pull/158))
 
+### Fixed
+- [Java] Altered workflow to use Java 17 LTS over Java 16 ([#167](https://github.com/cucumber/tag-expressions/pull/167))
```

Comment:
This doesn't change anything for the end user. So it doesn't have to be in the change log.

Reply:
Fixed with e60f74ae0764c52d549cbfe331d8c68fb42e9ebd

Reply:
We do document changes to GHA in our changelog. See the python change by Bjorn recently.

But given Java is your remit I see no reason to debate

Reply:
Mmh. The change logs is for notable changes. And I wouldn't consider changes like that notable as it doesn't change how the code works.